src/graphql/types/Mutation/createChat.ts

```
@@ -3,204 +1,151 @@
+            })),
+          },
+        });
+      }
+
+      const {
+        userIds: rawUserIds,
+        organizationId,
+        isGroup,
+        name,
+      } = parsed.data.input;
+      const me = ctx.currentClient.user!.id;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.

Unsafe fix: Replace with optional chain operator ?. This operator includes runtime checks, so it is safer than the compile-only non-null assertion operator


(lint/style/noNonNullAssertion)